src/main/java/io/fabric8/elasticsearch/plugin/OpenshiftRequestContextFactory.java

```diff
@@ -63,7 +64,7 @@ public OpenshiftRequestContextFactory(final Settings settings, final RequestUtil
                 ConfigurationSettings.DEFAULT_OPENSHIFT_OPS_PROJECTS);
         this.kibanaPrefix = settings.get(ConfigurationSettings.KIBANA_CONFIG_INDEX_NAME, ConfigurationSettings.DEFAULT_USER_PROFILE_PREFIX);
         this.kibanaIndexMode = settings.get(ConfigurationSettings.OPENSHIFT_KIBANA_INDEX_MODE, UNIQUE);
-        if(!ArrayUtils.contains(new String [] {UNIQUE, SHARED_OPS}, kibanaIndexMode.toLowerCase())) {
+        if(!ArrayUtils.contains(new String [] {UNIQUE, SHARED_OPS, SHARED}, kibanaIndexMode.toLowerCase())) {
```

Comment:
Hmm.. note to self there is an opportunity for this factory to rely on 'PluginSettings' so the call and initializations here are pushed into a common class.

Reply:
don't really know what you mean, I wanted to keep things identically